vector-app/src/androidTest/java/im/vector/app/ui/robot/space/SpaceCreateRobot.kt

```diff
@@ -85,7 +86,9 @@ class SpaceCreateRobot {
         clickOn(R.id.nextButton)
         waitUntilViewVisible(withId(R.id.recyclerView))
         clickOn(R.id.nextButton)
-        waitUntilDialogVisible(withId(R.id.inviteByMxidButton))
+        waitUntilActivityVisible<RoomDetailActivity> {
+            waitUntilDialogVisible(withId(R.id.inviteByMxidButton))
+        }
```

Comment:
Just to double check, is it OK that the change is not under `features.isNewAppLayoutEnabled()` condition?

Reply:
Yes, no matter of flag enabled/disabled that was failing for me locally without this change